### ISSUE TITLE
[Reputation][Mount discount]

### ISSUE THREAD

**Issue by martinmarkovic** [SallyWhitemane/Mograine#1078]
*Description:*
Getting from honored to revered in thrallmar did not reduce my flying skill cost from 4500 to 4250.
Also dude from support told me that exalted should discount flying from 5000 to 4250 but that is not the case... If exalted discount is 20% - 20% from 5000 is 1000 gold.
There for exalted flying should cost 4000 not 4250, thrallmar/Honorhold, Kirin Tor, Horde expedition/Ally Vanguard.

**How to reproduce:**

**How it should work:**

**Comment by scrublama** [comment]
yes it does have rep discount after patch 3.2.0. h̶t̶t̶p̶:̶/̶/̶w̶w̶w̶.̶w̶o̶r̶l̶d̶o̶f̶w̶a̶r̶c̶r̶a̶f̶t̶.̶c̶o̶m̶/̶i̶n̶f̶o̶/̶u̶n̶d̶e̶r̶d̶e̶v̶/̶i̶m̶p̶l̶e̶m̶e̶n̶t̶e̶d̶/̶3̶p̶2̶.̶x̶m̶l̶.̶
Neutral: 5000g
Friendly: 4750g
Honored: 4500g
Revered: 4250g
Exalted: 4000g

"As of the most recent patch, the base cost is now 250 gold, and the faction requirements have been **changed from Thrallmar to Orgrimmar.**

Blue post for reference: h̶t̶t̶p̶:̶/̶/̶f̶o̶r̶u̶m̶s̶.̶w̶o̶r̶l̶d̶o̶f̶w̶a̶r̶c̶r̶a̶f̶t̶.̶c̶o̶m̶/̶t̶h̶r̶e̶a̶d̶.̶h̶t̶m̶l̶?̶t̶o̶p̶i̶c̶I̶d̶=̶2̶0̶1̶3̶6̶5̶9̶5̶0̶9̶7̶&̶p̶a̶g̶e̶N̶o̶=̶2̶&̶s̶i̶d̶=̶1̶#̶2̶0̶
Last edited by devisevere on 2009/10/11 (Patch 3.2.2)"

**Comment by scrublama** [comment]
"yes it does have rep discount after patch 3.2.0. h̶t̶t̶p̶:̶/̶/̶w̶w̶w̶.̶w̶o̶r̶l̶d̶o̶f̶w̶a̶r̶c̶r̶a̶f̶t̶.̶c̶o̶m̶/̶i̶n̶f̶o̶/̶u̶n̶d̶e̶r̶d̶e̶v̶/̶i̶m̶p̶l̶e̶m̶e̶n̶t̶e̶d̶/̶3̶p̶2̶.̶x̶m̶l̶.̶
Neutral: 5000g
Friendly: 4750g
Honored: 4500g
Revered: 4250g
Exalted: 4000g."

"As of the most recent patch, the base cost is now 250 gold, and the faction requirements have been **changed from Thrallmar to Orgrimmar.**

Blue post for reference: h̶t̶t̶p̶:̶/̶/̶f̶o̶r̶u̶m̶s̶.̶w̶o̶r̶l̶d̶o̶f̶w̶a̶r̶c̶r̶a̶f̶t̶.̶c̶o̶m̶/̶t̶h̶r̶e̶a̶d̶.̶h̶t̶m̶l̶?̶t̶o̶p̶i̶c̶I̶d̶=̶2̶0̶1̶3̶6̶5̶9̶5̶0̶9̶7̶&̶p̶a̶g̶e̶N̶o̶=̶2̶&̶s̶i̶d̶=̶1̶#̶2̶0̶
Last edited by devisevere on 2009/10/11 (Patch 3.2.2)"
